itemManagement/views.py

```diff
@@ -18,14 +18,15 @@ class HomePage(SearchView):
 
     def get_queryset(self):
         queryset = super(HomePage, self).get_queryset()
-        return queryset.filter(deleted=False)
+        return queryset.filter(deleted=False).order_by('-lastUsed')
 
     def get_context_data(self, *args, **kwargs):
         context = super(HomePage, self).get_context_data(*args, **kwargs)
         print(context)
         objectList = context['object_list']
         if (len(objectList) == 0):
-            context['items'] = [item.getItemSummary() for item in Item.objects.filter(deleted=False).all()]  # Todo, return list of most recently used items
+            context['items'] = [item.getItemSummary() for item in Item.objects.filter(
+                deleted=False).order_by('-lastUsed', 'title')]
```

Comment:
I highly suggest against returning every single item there is including one image per item (at least 400 items with 400 images). Have you tried uploading 400 images and seeing the response time from a remote server